### DISCUSSION TITLE
Do the monetary emission factors include taxes?

### DISCUSSION THREAD

**Discussion by nanocate** [USEPA/supply-chain-factors#18] [Q&A] [answered]
Hello! 
I was wondering if the emission factors of the EPA US EEIO Supply Chain and the EPA US EEIO include taxes or not? 
Meaning is the unit kgCO2e/$ (excluding/including VAT)?
Thanks for the clarification!

**Comment by MoLi7** [comment, marked as answer]
Hi @nanocate, thanks for you question. I want to correct my previous answer.

In fact, according to the BEA's Input-Output Handbook (Concepts and Methods of the U.S. Input-Output Accounts), **commodity taxes ARE included in the producers' value (PRO) of commodity output**. See [Chapter 8 of the Handbook](https://www.bea.gov/sites/default/files/methodologies/IOmanual_092906.pdf#page=155).

> Commodity taxes—which consist of sales and excise taxes—are part of commodity output and are included in the producers’ value of transactions. These taxes are included in producers’ prices because they are collected by the producer and are paid by the purchaser. Because the use table shows all transactions in producers’ prices, commodity taxes are not shown separately. Commodity taxes are collected from purchasers by businesses on the behalf of government and so, in business accounting, they do not represent a direct business expense of the firm. Thus, they differ from some of the other types of taxes, such as property taxes, that constitute a direct business expense. In the I-O use table, all of these taxes are included in the row “taxes on production and imports less subsidies.”

The taxes are also included in the purchasers' value (PUR) of commodity output, because the conversion from PRO to PUR is just adding margin costs.

Therefore, both the emission factors from the Supply Chain GHG models (PUR) and those from the USEEIO models (in PRO) include taxes in their units.

**Reply by nanocate** [reply]
Thanks, @MoLi7 and @WesIngwersen for your answers. 
This is the case for the purchaser price. How about the unit of the emission factors for the producer price? i.e. the factors in the [USEEIO matrices](https://catalog.data.gov/dataset/useeio-v1-1-matrices)?

**Comment by WesIngwersen** [comment]
Just adding to this thread that the [US doesn't have a Value-Added Tax (VAT)](https://www.taxpolicycenter.org/briefing-book/what-vat), and therefore since the Factors are based on the US economy, there is no VAT impact included.

**Reply by mkamperhoff** [reply]
Hi everyone, 

I trust this message finds you well. I am reaching out to seek guidance on a matter related to the use of EPA's spend-based emission factors for calculations beyond the scope of the US economy.

Specifically, my query pertains to whether calculations should consider the amount spent on a given commodity X inclusive or exclusive of taxes when utilizing EPA's emission factors in an international context.

Theoretically, considering the inclusion of commodity taxes in the base model (BEA's Input-Output Model), one might conclude that taxes should be factored into the calculations. However, the absence of VAT in the US implies a lack of tax consideration. Consequently, one might argue for the exclusion of taxes in the calculations.

I would greatly appreciate any recommendations or insights you may have on this matter. While I acknowledge that these factors were not specifically designed for use outside the US, I am exploring their application for proxy calculations.

Thank you for your attention and assistance.

Kind regards 
Max

**Reply by WesIngwersen** [reply]
Just to repeat that taxes are included in the price used in the Factors. Tax here does not affect the numerator in the Factor, it just increased the denominator. The objective with the denominator is to be relevant for a price paid by tour consumer. If VAT taxes are contributing to price, those need to be included in the calculation to derive the denominator, as this is reflective of purchaser price.
